packages/client/src/components/collection/CollectionsContainer.tsx

```diff
@@ -0,0 +1,51 @@
+import { clsx } from "clsx";
+import React, { useState } from "react";
+
+import { CollectionListItem } from "@/components/collection/CollectionListItem";
+import { Collection as CollectionEntity } from "@/types/collection.type";
+import SearchField from "@/components/ui/SearchField";
+
+interface Props {
+  collections: CollectionEntity[];
+}
+
+export default function CollectionsContainer({ collections }: Props) {
+  const [filteredCollections, setFilteredCollection] = useState(collections);
+
+  function Filter(searchPhrase: string) {
+    if (searchPhrase == "") {
+      setFilteredCollection(collections);
+    } else {
+      setFilteredCollection(
+        collections.filter((cur) => {
+          return cur.name.toLowerCase().includes(searchPhrase);
+        })
```

Comment:
Így is teljesen jó, de lehet kicsit egyszerűbben is:

`collections.filter((cur) => cur.name.toLowerCase().includes(searchPhrase))`